tests/openssl_moretest/enc/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+set(EDL_FILE ../openssl_moretest.edl)
+
+add_custom_command(
+  OUTPUT openssl_moretest_t.h openssl_moretest_t.c
+  DEPENDS ${EDL_FILE} edger8r
+  COMMAND
+    edger8r --trusted ${EDL_FILE} --search-path ${PROJECT_SOURCE_DIR}/include
+    ${DEFINE_OE_SGX} --search-path ${CMAKE_CURRENT_SOURCE_DIR})
+
+add_enclave(
+  TARGET
+  openssl_moretest_enc
+  CXX
+  UUID
+  90c020e8-e5dd-11ea-adc1-0242ac120002
+  SOURCES
+  enc.cpp
+  tests_ecalls.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/../common_code/common_digest_tests.cpp
+  ${CMAKE_CURRENT_BINARY_DIR}/openssl_moretest_t.c)
+
+enclave_compile_options(
+  openssl_moretest_enc PRIVATE -Wno-shorten-64-to-32 -Wno-sign-conversion
+  -Wno-conversion -Wno-unused-parameter)
+
+enclave_include_directories(
+  openssl_moretest_enc
+  PRIVATE
+  ${OPENSSL_DIR}/../..
```

Comment:
You can simplify this by simply including
```
${CMAKE_CURRENT_BINARY_DIR}
${OPENSSL_DIR}
${OPENSSL_DIR}/include
${CMAKE_BINARY_DIR}/3rdparty/openssl/build/include
```
Then you don't need `${OPENSSL_DIR}/crypto/*` below

Reply:
You are right. I have updated both host/CMakeLists.txt and enc/CMakeLists.txt in this way.